k9mail/src/main/java/com/fsck/k9/mailstore/MessageViewInfoExtractor.java

```diff
@@ -20,6 +19,7 @@
 import com.fsck.k9.mail.internet.Viewable;
 import com.fsck.k9.message.extractors.AttachmentInfoExtractor;
 import com.fsck.k9.ui.crypto.MessageCryptoAnnotations;
+import com.fsck.k9.ui.crypto.MessageCryptoSplitter;
 import com.fsck.k9.ui.crypto.MessageCryptoSplitter.CryptoMessageParts;
```

Comment:
No need to fix this right away. But "backend" code referencing "UI" code is always an error. In this case `MessageCryptoSplitter` and friends should probably not live in the UI package. If there's a real dependency on some UI-related code there's an abstraction missing.